torch/masked/maskedtensor/core.py

```diff
@@ -571,7 +611,16 @@ def to_tensor(self, value):
         return self.get_data().masked_fill(~self.get_mask(), value)
 
     def get_data(self):
-        return self._masked_data
+        class GetData(torch.autograd.Function):
```

Comment:
What does the masked out values contain here? Can they be anything?

Reply:
If you mean `self._masked_data`, really the only restrictions are the ones in `_validate_members()` -- e.g. we only support a few different data layouts, only floating point / bool / int dtypes, etc.